playwright/fixtures/widget-user.ts

```diff
@@ -113,7 +113,8 @@ async function registerUser(
   await page.getByRole("button", { name: "Register" }).click();
   const continueButton = page.getByRole("button", { name: "Continue" });
   try {
-    await expect(continueButton).toBeVisible({ timeout: 5000 });
+    await expect(continueButton).toBeVisible({ timeout: 700 });
+    // why do we need to put in the passwor if there is a continue button?
```

Comment:
I think this can be removed now? Looks like it was some old onboarding UI from web that is not needed anymore?

Reply:
remove all `continueButton` related code.